core/issuers.js

```diff
@@ -35,6 +65,21 @@ Description must be truthy: ${description}`;
       getBalance() {
         return mintController.getAmount(payment);
       },
+      getUse() {
+        return makeUseObj(issuer, payment);
+      },
+      getDelegatedUse() {
+        // quantity is the same, but amounts are different for
+        // different issuers
+        const childAmount = getChildAmount(amount);
+        const childPurse = childMint.mint(childAmount);
+        const childPayment = childPurse.withdrawAll();
+        return makeUseObj(childIssuer, childPayment);
+      },
+      revokeChildren() {
+        const childAmount = getChildAmount(amount);
+        childMint.destroy(childAmount);
```

Comment:
We have co-existing terminology `destroy`, `revoke`, and `burn`. Are they three distinct concepts? Can we consolidate some of these concepts? Can we explain their distinct meanings?

Reply:
I got rid of `destroy` so now we only have `revoke` and `burn`. `Burn` is a consensual handing over of rights that are made inaccessible, and 'revoke' is nonconsensual destroying of rights.